src/en/articles/landmarks/index.njk

```diff
@@ -1,136 +1,236 @@
+
+  <h4 id="footer">footer</h4>
+  <p>The <code>footer</code> tag (or the <abbr>ARIA</abbr> landmark attribute <code>role="contentinfo"</code>) is used to identify the footer. This is only true if this element is placed in a <code>body</code> tag,
+      either as a direct child, or in an <abbr>HTML</abbr> tag other than <code>article</code>, <code>aside</code>, <code>main</code>, <code>nav</code> and <code>section</code>.</p>
+  <p>To simplify things, we recommend using the <code>footer</code> tag (or the <abbr>ARIA</abbr> landmark attribute <code>role="contentinfo"</code>) only once per page.</p>
+  <p>Note: if an HTML page contains an <code>iframe</code>, it can also contain a <code>main</code> tag (or a <code>role="main"</code> attribute) since it's a stand-alone page.</p>
 
-<p>All these landmarks can rely on the <abbr>ARIA</abbr> attributes that are <code>aria-label</code>, <code>aria-labelledby</code> and <code>aria-describedby</code> to provide them with a more relevant accessible name.</p>
-<p> Especially when several landmarks of the same type are present in the page, it may be important to give them a description via these <abbr>ARIA</abbr> attributes. </p>
-<p>Example of a page with three navigation regions: </p>
-<pre><code class = "html">
- &lt;nav aria-label="Main Menu"&gt; [...] &lt;/nav &gt;
- &lt;nav aria-label="Secondary Menu"&gt; [...] &lt;/nav&gt;
- &lt;nav aria-label="breadcrumb trail"&gt; [...] &lt;/nav&gt;
-</code></pre>
-<p>More details on <a href="../../articles/aria-attributes-that-can-save-you/"> "The <abbr>ARIA</abbr> attributes that can save you" </a>.</p>
 
-<p>Another best practice is that all content must be included in the Landmarks / <abbr>HTML</abbr>5 structure tags. This allows users to not forget content that they would not have found. </p>
+<h3 class="mt-4" id="best_practices">Best practices</h3>
+<ul>
+  <li>
+    <p class="mb-1">When several landmarks of the same type are present on the page, it may be appropriate to give each one a different accessible name to enable technical support users to differentiate between them. An accessible name is added to:</p>
+    <ul>
+      <li>an <code>aria-labelledby</code> attribute pointing to the header title whose content you want to use as an accessible name, if the region has a header title.</li>
+      <li>an <code>aria-label</code> attribute, if the region doesn't have a header title (e.g., text in an <code>h1</code> to <code>h6</code> tag).</li>
+    </ul>
+    <p class="mt-2"><i>Note: </i>an <code>aria-describedby</code> attribute, pointing to an element whose content you want to use as a complement to the accessible name (accessible description), can also be used to specify the function of a region.</p>
+ 
+    <p class="mt-3>Example of a page with three navigation landmarks:</p>
```

Comment:
```suggestion
    <p class="mt-3">Example of a page with three navigation landmarks:</p>
```

Reply:
Could explain why this text does not appear in the page:
<img width="1199" height="154" alt="image" src="https://github.com/user-attachments/assets/d4afa9cf-a0b1-490e-899a-0d6419a4eae6" />